charts/kuberpult/templates/frontend-service.yaml

```diff
@@ -118,6 +118,8 @@ spec:
           value: "https://{{ .Values.ingress.domainName }}"
```

Comment:
please change the commit messages, e.g.
`feat: implement endpoint to retrieve manifests` (and then more details in the next line)

Reply:
done